iina/CollapseView.swift

```diff
@@ -55,7 +55,7 @@ class CollapseView: NSStackView {
     updateContentView(animated: animated)
   }
 
-  @objc private func triggerAction(_ sender: NSControl) {
+  @objc private func triggerAction() {
```

Comment:
You can't remove this parameter, see https://developer.apple.com/library/archive/documentation/General/Conceptual/Devpedia-CocoaApp/TargetAction.html#//apple_ref/doc/uid/TP40009071-CH3-SW2

Reply:
It's [legal on iOS](https://developer.apple.com/documentation/uikit/uicontrol#1658488), but not on macOS. Things will mostly still work because the argument register will probably not get touched but it's good to do it correctly.